json/flagd-definitions.yaml

```diff
@@ -1,39 +1,36 @@
----
-"$schema": http://json-schema.org/draft-07/schema#
+$schema: http://json-schema.org/draft-07/schema#
 title: flagd Flag Configuration
```

Comment:
```suggestion
title: Flagd Flag Configuration
```

Reply:
no

Reply:
Even in a title? We need a naming guide so we can be consistent.

Reply:
I'd be happy to write one.

But ya, IMHO, stylized names like "flagd" should not be capitalized, even at the start of a sentence. See etcd docs: https://etcd.io/docs/v3.5/learning/design-client/#introduction